ext/phook_observer.c

```
@@ -1,36 +1,36 @@
 
 #include "php.h"
-#include "otel_observer.h"
+#include "phook_observer.h"
 #include "zend_observer.h"
 #include "zend_execute.h"
 #include "zend_extensions.h"
 #include "zend_exceptions.h"
 #include "zend_attributes.h"
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If ZEND_BEGIN_MODULE_GLOBALS is a macro then please configure it.

(unknownMacro)